docs/output.md

```diff
@@ -546,28 +555,26 @@ For all samples:
 **Output directory: `results/Reports/[SAMPLE]/bamQC`**
 
 - `VariantCaller_[SAMPLE].bcf.tools.stats.out`
-  - RAW statistics used by MultiQC
-
-For more information about how to use Qualimap bamqc reports, see [Qualimap bamqc manual](http://qualimap.bioinfo.cipf.es/doc_html/analysis.html#id7)
+  - Raw statistics used by `MultiQC`
 
-#### MarkDuplicates reports
+For further reading and documentation see the [Qualimap bamqc manual](http://qualimap.bioinfo.cipf.es/doc_html/analysis.html#id7)
 
-[[GATK MarkDuplicatesSpark](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.php), Spark implementation of [Picard MarkDuplicates](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/picard_sam_markduplicates_MarkDuplicates.php) locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+#### GATK MarkDuplicates reports
 
```

Comment:
General comment: In the `preprocessing` folder, the files that ran through `MarkDuplicates` live in a folder called `DuplicatesMarked`, whereas here they are in a directory called `MarkDuplicates`. Might make sense to unify the naming at some point. What do you think?

Reply:
Let's think about that for DSL2 ;-)